addons/website/views/snippets/s_countdown.xml

```diff
@@ -17,7 +17,7 @@
 <template id="s_countdown_options" inherit_id="website.snippet_options">
     <xpath expr="." position="inside">
         <div data-js="countdown" data-selector=".s_countdown">
-            <we-datetimepicker string="Due Date" data-select-data-attribute="" data-attribute-name="endTime"/>
+            <we-datetimepicker string="Due Date" data-select-data-attribute="" data-attribute-name="endTime" data-format="X"/>
```

Comment:
@rdeodoo Here I make some changes in the countdown. More explanation in the commit message ! Can you validate plz :)

Reply:
I guess it makes sense. Maybe there is a small problem: seeing the code my guess is that the date format of the datepicker UI is equal to the dateformat the widgets will provide to the option, it should not. It should be displayed in the left panel using the user date format but should send the value in website lang date format. Check if easy to do, otherwise I guess we can forget about it.

Reply:
That's why I used 'X'

Reply:
It raises the same warning, regardless of the uppercase, to check if this is problematic, it seems to be listed in their up to date doc so it is weird

Reply:
@fja-odoo You marked the PR as ready again but I don't see any change to support date correct following @rdeodoo concerns?